### PR TITLE
page size support + fix orientation

### PR DESCRIPTION
**Pull request by Bars92**
calculate width and height by orientation and added support for paper size.

### CODE REVIEW

File: reportsServer.js [hunk 1]
```diff
@@ -23,6 +28,16 @@ function getPageSize(pageSize) {
   }
 }
 
+function getPageSizeByOrientation(pageSize, orientation) {
+  const size = getPageSize(pageSize);
+  if (orientation && orientation === PAGE_ORIENTATION.landscape) {
+    const h = size.height;
+    const w = size.width;
+    return { width: h, height: w };
```
Comment (dorsha) [inline reportsServer.js]: shouldn't it be the opposite? you assigned h to width and w to height..

Comment (dorsha) [reply reportsServer.js]: and no need for the h, w consts..

Comment (Bars92) [reply reportsServer.js]: if the orientation is landscape then the width and height are switched.
image
cant without the consts..

Comment (dorsha) [reply reportsServer.js]: 👍

**Pull request merged**